lib/GdbCrashHandlerDialog.cpp

```diff
@@ -30,7 +30,7 @@
 #include <QTimer>
 #include <QUrlQuery>
 #include <QUuid>
-#include <quazipfile.h>
+#include <quazip/quazipfile.h>
```

Comment:
This won't work as a general upstream solution, i.e. other distros use quazip5/quazipfile.h. The proper way is for quazip-devel to ship a CMake module and/or pkg-config file which returns the include path. On Fedora, FindQuaZip5.cmake does this, but quazip does not seem to provide a pkg-config file (which is the typical way to include dependencies with qmake). You'll need to explicitly pass the include dir via QMAKE_CFLAGS env-var when building the project, or patch it locally.